lib/matplotlib/colorbar.py

```diff
@@ -742,12 +742,12 @@ def _set_label(self):
     def set_label(self, label, *, loc=None, **kwargs):
         """Add a label to the long axis of the colorbar."""
         _pos_xy = 'y' if self.orientation == 'vertical' else 'x'
-        _protected_kw = [_pos_xy, 'horizontalalignment', 'ha']
-        if any([k in kwargs for k in _protected_kw]):
+        protected_kw = ['x', 'horizontalalignment', 'ha']
```

Comment:
Please do not mix fixes/semantic changes into large code cleanup PRs. This makes reviewing harder and it's basically impossible to find this when browsing through commit messages.

Reply:
I split out that part (which should have been done in https://github.com/matplotlib/matplotlib/pull/16274) into a separate first commit in the PR (one needs to at least fix the indent below anyways).